src/utils.js

```diff
@@ -232,7 +240,8 @@ export const isItemSoldInShop = ({ id }) => Boolean(shopInventoryMap[id])
  * @param {farmhand.item} item
  * @returns {number}
  */
-export const getResaleValue = ({ id }) => itemsMap[id].value / 2
+export const getResaleValue = ({ id }, completedAchievements) =>
```

Comment:
It looks like we can drop the `completedAchievements` parameter here.

Reply:
oops! yep, didn't end up going that route. removing